modules/UI/videolayout/LocalVideo.js

```diff
@@ -121,9 +122,10 @@ LocalVideo.prototype.changeVideo = function(stream) {
 
     // eslint-disable-next-line eqeqeq
     const isVideo = stream.videoType != 'desktop';
+    const settings = APP.store.getState()['features/base/settings'];
```

Comment:
I absolutely agree. Not sure though if you meant it for settings only, or generally a solution to hide all redux tree names? Anyhow, I added it for settings for now, but yeah, a more general solution would be awesome. Question though, if abstracting it will make it simpler or more complex.

Reply:
I was referring to selectors being a general practice so that the outside world don't need to do the key lookup on the store. But yeah, I'm wondering the same thing about making it more simpler or complex. It would make it simpler in certain situations where state has to be derived from multiple places in redux and it could hide changes that occur in the store structure, but for the common case it may be too much to keep the consistency. I'm surprised you went ahead and made the change for this PR; I actually wasn't expecting you to and if it sounded like I was asking it be done for this PR then I wrote my message incorrectly. Since you've done it, how do you feel about the selector for this changeset? It seems like it carries the same question: is it making it more complex or simpler?